pom.xml

```diff
@@ -450,6 +460,11 @@
                 <artifactId>snakeyaml</artifactId>
                 <version>${snakeyaml.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.json</groupId>
+                <artifactId>json</artifactId>
+                <version>${json.version}</version>
```

Comment:
We already have Jackson as the library for JSON. Is there a special reason to use org.json for your feature? If not, sticking to Jackson is better imo

Reply:
JSON was used to parse a JSON payload receiving from Jenkins API. Changed it as to use existing jackson library and removed JSON library.